app/controllers/concerns/idv_step_concern.rb

```diff
@@ -17,9 +17,18 @@ def increment_step_attempts
     idv_session.step_attempts[step_name] += 1
   end
 
+  def show_vendor_fail
+    @view_model = Object.const_get("#{step_name.capitalize}New").new(modal: 'fail')
+    @presenter = VerificationPresenter.new(step_name, @view_model.modal_type)
```

Comment:
would it be hard to derive this presenter from within `view_model` ? Looks like we'd need to pass `step_name` to the view model, which would't be too bad. That way, we could stick with just one instance variable.